### PR TITLE
Remove duplicated sections in Fluentd tutorial

### LINKED ISSUE #589: "Deploy a fluentd" duplicate sections
In [deploy/application/docker/tutorials/fluentd.md](https://github.com/CleverCloud/doc.clever-cloud.com/blob/d97a261028934dff87d8a1dcaa1f1b5685203102/deploy/application/docker/tutorials/fluentd.md), `{{< readfile "/content/partials/create-application.md" >}}` and `{{< readfile "/content/partials/set-env-vars.md" >}}` appear twice.

Screenshot of 'Deploy a fluentd' duplicate sections

I'll remove them soon, and check if the problem appears in other files too.

### PR DESCRIPTION
**Pull request by gaelreyrol**
Close #589.

### CODE REVIEW

File: deploy/application/docker/tutorials/fluentd.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ Since you deploy microservices on Clever Cloud, you may need some data pipes bet
 
 Fluentd is an open source data collector written in Ruby, which lets you unify the data collection and consumption for a better use and understanding of data.
 
+{{< alert "info" "Why I can't just use Ruby runtime ?" >}}
```
Comment (BlackYoup) [inline deploy/application/docker/tutorials/fluentd.md:26]:
```suggestion
{{< alert "info" "Why can't I use the Ruby runtime ?" >}}
```

I had a hard time figuring out that the "ruby runtime" here is a ruby application. Maybe we could replace `the Ruby runtime` by `a Ruby application`? I don't know

File: deploy/application/docker/tutorials/fluentd.md [hunk 2]
```diff
@@ -23,11 +23,16 @@ Since you deploy microservices on Clever Cloud, you may need some data pipes bet
 
 Fluentd is an open source data collector written in Ruby, which lets you unify the data collection and consumption for a better use and understanding of data.
 
+{{< alert "info" "Why I can't use the Ruby application ?" >}}
```
Comment (BlackYoup) [inline deploy/application/docker/tutorials/fluentd.md:26]:
```suggestion
{{< alert "info" "Why can't I use the Ruby application?" >}}
```

**Review by BlackYoup — CHANGES_REQUESTED** [review]
Last small modification but other than that, ok :+1:

File: deploy/application/docker/tutorials/fluentd.md (hunk 2 above)
Comment (gaelreyrol) [reply deploy/application/docker/tutorials/fluentd.md]: Fixed

**Review by BlackYoup — APPROVED** [review]

**Pull request merged**